app/src/shared/consts/index.ts

```diff
@@ -0,0 +1,66 @@
+import { z } from 'zod'
+
+export const GraphProject = z.object({
```

Comment:
This is the new `GraphProject`. It's zod-driven. I'd like us to stick to it, and derive types from it. See `z.infer<typeof GraphProject>` and `GraphProject.shape`.